Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -257,60 +266,50 @@ private boolean validateIdsInServerExtras(Map<String, String> serverExtras) {
         return isAllDataValid;
     }
 
-    @NonNull
-    public String getAdNetworkId() {
-        return mPlacementId != null ? mPlacementId : "";
-    }
-
-    @Override
-    protected boolean checkAndInitializeSdk(@NonNull final Activity activity,
-                                            @NonNull final AdData adData) {
-        return false;
-    }
-
-    @Nullable
-    public View getAdView() {
-        return mAdView;
+    private String getAdNetworkId() {
+        return mPlacementId;
     }
 
     private class VungleBannerRouterListener implements VungleRouterListener {
 
         @Override
-        public void onAdEnd(String placementId) {
-            if (mPlacementId.equals(placementId)) {
-                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdEnd placement id: " + placementId);
+        public void onAdEnd(@NonNull String placementReferenceId) {
+            MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdEnd placement id" + placementReferenceId);
+            if (mPlacementId.equals(placementReferenceId)) {
+                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdEnd - Placement ID: " +
+                        placementReferenceId);
                 mIsPlaying = false;
                 sVungleRouter.removeRouterListener(mPlacementId);
                 mVungleRouterListener = null;
-            }
-        }
-
-        @Override
-        public void onAdClick(String placementId) {
-            if (mPlacementId.equals(placementId)) {
-                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdClick placement id: " + placementId);
                 mHandler.post(new Runnable() {
 
                     @Override
                     public void run() {
-                        if (mInteractionListener != null) {
-                            mInteractionListener.onAdClicked();
+                        if (mCustomEventBannerListener != null) {
+                            mCustomEventBannerListener.onBannerClicked();
```

Comment:
we can't call mCustomEventBannerListener.onBannerClicked() in any case, there were params in onAdEnd() but it call was moved to onAdClick()